core/urls.py

```diff
@@ -22,6 +22,10 @@
     path('operations/data/<int:pk>/', core_views.ViewData.as_view()),
     path('themes/', core_views.ThemeList.as_view()),
     path('themes/<int:pk>/', core_views.ThemeDetail.as_view()),
+    path('schedule_event/', core_views.ScheduledEventList.as_view()),
+    path('scheduled_event_run_instance_list/', core_views.ScheduledEventRunInstanceList.as_view()),
+    path('scheduled_event_run_instance_list/<int:pk>/', core_views.ScheduledEventRunInstanceDetail.as_view()),
+    path('scheduled_event_run_instance_list/history/<int:schedule_event_id>/', core_views.ScheduledEventRunInstanceHistory.as_view()),
```

Comment:
`scheduled_event/<int:pk>/run_instances` - to fetch all instances for a particular event & add an instance (GET, POST)
`scheduled_event/run_instance/<int:pk>/` - to handle a particular instance (GET, PUT) - we never delete

Reply:
Updated according to comment